README.md

```diff
@@ -19,19 +19,20 @@ English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.
 
 # QuestDB
 
-QuestDB is a high-performance, open-source SQL database for applications in
-financial services, IoT, machine learning, DevOps and observability. It includes
-endpoints for PostgreSQL wire protocol, high-throughput schema-agnostic
-ingestion using InfluxDB Line Protocol, and a REST API for queries, bulk
-imports, and exports.
-
-QuestDB implements ANSI SQL with native extensions for time-oriented language
-features. These extensions make it simple to correlate data from multiple
-sources using relational and time series joins. QuestDB achieves high
-performance from a column-oriented storage model, massively-parallelized vector
-execution, SIMD instructions, and various low-latency techniques. The entire
-codebase was built from the ground up in Java and C++, with no dependencies, and
-is 100% free from garbage collection.
+QuestDB is an open-source time-series database for high throughput ingestion and
+fast SQL queries with operational simplicity. It supports schema-agnostic
+ingestion using the InfluxDB line protocol, PostgreSQL wire protocol, and a REST
+API for bulk imports and exports.
+
+QuestDB is well suited for financial market data, application metrics, sensor
+data, real-time analytics, dashboards, and infrastructure monitoring.
+
+QuestDB uses ANSI SQL and benefits from time-series SQL semantics. It is simple
+to correlate data from multiple sources using relational and time-series joins.
```

Comment:
QuestDB uses ANSI SQL and benefits from time-series SQL semantics. It is simple
to correlate data from multiple sources using relational and time-series joins.

I would change to

QuestDB implements ANSI SQL with native time-series SQL semantics. These semantics 
make it easy to correlate data from multiple sources using relational and time-series joins.